src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -98,6 +106,16 @@ public static ProviderCultureResult ParseCookieValue(string value)
             var cultureName = potentialCultureName.Substring(_culturePrefix.Length);
             var uiCultureName = potentialUICultureName.Substring(_uiCulturePrefix.Length);
 
+            if (!IsValidCulture(cultureName))
+            {
+                _logger.ParsedCulture(nameof(CookieRequestCultureProvider), cultureName);
+            }
+
+            if (!IsValidCulture(uiCultureName))
+            {
+                _logger.ParsedCulture(nameof(CookieRequestCultureProvider), uiCultureName);
```

Comment:
These two logs are identical in terms of the message - perhaps they should have some distinction to indicate which culture was being parsed, i.e. Culture vs. UICulture?